src/MenuGraphr.ts

```diff
@@ -721,6 +724,19 @@ export class MenuGraphr implements IMenuGraphr {
     }
 
     /**
+     * Clears saved indices that should be forgotten when the menu is deleted.
+     * 
+     * @param name   The name of the menu that is being deleted.
+     */
+    public clearMenuIndices(name: string): void {
+        const menu: IMenu = this.menus[name];
```

Comment:
Use `this.getExistingMenu(name)` for public functions. Trying to clear indices of an unknown menu will then throw a better error.